alves/style-rtl.css

```diff
@@ -2770,6 +2770,12 @@ table.is-style-stripes tbody tr:nth-child(odd),
  * Components
  * - Similar to Blocks but exist outside of the "current" editor context
  */
+.site-logo a {
+	display: inline-block;
+	width: 64px;
```

Comment:
Should we use max-width instead in case the user has a smaller logo? We don't want to stretch them, do we?